blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java

```diff
@@ -2291,7 +2292,12 @@ public void parameterizedPipeline() throws Exception {
         Assert.assertEquals("param2", parameters.get(1).get("name"));
         Assert.assertEquals("StringParameterDefinition", parameters.get(1).get("type"));
         Assert.assertEquals("string param", parameters.get(1).get("description"));
-        Assert.assertNull(((Map) parameters.get(1).get("defaultParameterValue")).get("value"));
+
+        if (Jenkins.getVersion().isOlderThan(new VersionNumber("2.281"))) {
+            Assert.assertNull(((Map) parameters.get(1).get("defaultParameterValue")).get("value"));
+        } else {
+            Assert.assertEquals("", ((Map) parameters.get(1).get("defaultParameterValue")).get("value"));
+        }
```

Comment:
or

```suggestion
        Assert.assertNull(Util.fixEmpty(((Map) parameters.get(1).get("defaultParameterValue")).get("value")));
```

Reply:
I wanted to leave the difference clear. To get rid of what's going to happen with the core PR, this approach is better.